src/main/webapp/script.js

```diff
@@ -110,4 +110,82 @@ function getLoginStatus() {
       loginStatusElement.appendChild(textElement);
     }
   });
+
+}
+
+// test function for displaying recipes
+function hardCodedRecipeCard() {
+  const displayRecipeElement = document.getElementById('display-recipes');
+  displayRecipeElement.innerHTML = "";
+  displayRecipeElement.appendChild(createRecipeElement());
+}
+
+/** Creates an element that represents a recipe */
+// currently only has hardcoded values
+function createRecipeElement() {
+  const recipeElement = document.createElement('div');
+  recipeElement.className = 'recipe-card';
+
+  const titleElement = document.createElement('p');
+  titleElement.innerText = "Title";
+  recipeElement.appendChild(titleElement);
+
+  const infoElements = document.createElement('div');
+  infoElements.className = 'recipe-card-block';
+
+  const imageElement = document.createElement('img');
+  imageElement.src = "/images/salad.jpeg";
+  infoElements.appendChild(imageElement);
+
+  const linkElement = document.createElement('a');
+  linkElement.href = "https://www.w3schools.com/jsref/dom_obj_image.asp";
```

Comment:
is this link intended?

Reply:
This link is a hard-coded link for testing.